site/reference/cli/fn/eval/README.md

```diff
@@ -105,6 +105,13 @@ fn-args:
   If enabled, container functions are allowed to access network.
   By default it is disabled.
 
+--output, o:
+  If specified, the output resources are written to provided location.
```

Comment:
describe the behavior when not specified.

Reply:
updated.